### ISSUE TITLE
Incorrect arrow position in tooltip

### ISSUE THREAD

**Issue by Siegrift** [api3dao/api3-dao-dashboard#197]
![tooltip arrow position](https://user-images.githubusercontent.com/22679154/124498392-8df27780-ddbc-11eb-8225-218042dc7b20.png)

**Comment by andreogle** [comment]
This is a weird one. I haven't noticed this happen once myself but Tamara has reported it happening to her too. I'm using Firefox 89.0.2.

Does it correct itself eventually?

**Comment by Siegrift** [comment]
No, it can happen reliably. You need to have devtools open such that there is less horizontal space. You can't reproduce this?

**Comment by andreogle** [comment]
Ah yeah I can reproduce now by reducing the horizontal space. I'm guessing this is a problem with `rc-tooltip`, which I've noticed is quite buggy.

**Comment by Siegrift** [comment]
No longer relevant.
